stable/etcd-operator/templates/operator-deployment.yaml

```diff
@@ -0,0 +1,55 @@
+{{- if .Values.deployments.etcdOperator }}
+---
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "etcd-operator.fullname" . }}
+  labels:
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    app: {{ template "etcd-operator.fullname" . }}
```

Comment:
The `app` label should be `{{ template "etcd-operator.name" . }}`. I know the existing chart already used the `fullname` template, but since you pretty much rework the whole chart, I'd suggest you change this.